x/tradeshield/keeper/msg_cancel_spot_orders.go

```diff
@@ -0,0 +1,34 @@
+package keeper
+
+import (
+	"context"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
+	"github.com/elys-network/elys/x/tradeshield/types"
+)
+
+func (k msgServer) CancelSpotOrders(goCtx context.Context, msg *types.MsgCancelSpotOrders) (*types.MsgCancelSpotOrdersResponse, error) {
```

Comment:
@cryptokage1996 can we move this function to `msg_server_pending_spot_orders.go` similarely to perpetual? thanks